src/main/java/com/Incubatormsrv/Incubatormsrv/weather/mapper/WeatherMapper.java

```diff
@@ -0,0 +1,18 @@
+package com.Incubatormsrv.Incubatormsrv.weather.mapper;
+
+import _Users_luyanda_glucode.com_Documents_GitHub_Incubator_msrv.model.WeatherResponse;
+import com.Incubatormsrv.Incubatormsrv.weather.model.WeatherModel;
+import org.mapstruct.Mapper;
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
@Luyanda-Glucode Sorry man, not sure why you're manually defining MSRV controller response models... Your yaml should generate API models for 3rd party and then a models and controller definitions your your endpoint. 

this class then maps between the two models.